src/wallet/coinselection.h

```diff
@@ -166,6 +166,18 @@ struct OutputGroup
     CAmount GetSelectionAmount() const;
 };
 
+/** Compute the waste for this result given the cost of change
+ * waste = change_cost + excess + inputs * (effective_feerate - long_term_feerate)
+ * excess = selected_target - actual_target
```

Comment:
As pointed out above, this formula would count the input fees again, and here also the change cost.

Reply:
Changed to `selected_effective_value`.